vignettes/shiny-react.Rmd

```diff
@@ -78,54 +76,53 @@ Remember - components are functions which take props and return elements:
 ```r
 component <- function(name) {
   function(...) shiny.react::reactElement(
-    module = "@blueprintjs/core",
+    module = "@emdgroup-liquid/liquid",
     name = name,
     props = shiny.react::asProps(...),
-    deps = blueprintDependency()
+    deps = liquidDependency()
   )
 }
 ```
 
-We can now add Blueprint components to our package easily!
-Let's try a [Switch](https://blueprintjs.com/docs/#core/components/switch)
-and a [ProgressBar](https://blueprintjs.com/docs/#core/components/progress-bar) for starters.
+We can now add Liquid components to our package easily!
+Let's try a [LdButton](https://emdgroup-liquid.github.io/liquid/components/ld-button/)
+and a [LdLoading](https://emdgroup-liquid.github.io/liquid/components/ld-loading/) for starters.
 ```r
 #' @export
-ProgressBar <- component("ProgressBar")
+LdButton <- component("LdButton")
 
 #' @export
-Switch <- component("Switch")
+LdLoading <- component("LdLoading")
 ```
 
-### Adding Blueprint
-In the `js` directory we use `yarn` to add the Blueprint library.
-The [documentation](https://blueprintjs.com/docs/#blueprint.quick-start)
-also suggests adding `react` and `react-dom`,
-but we skip them as they are already provided by shiny.react.
+### Adding Liquid
+In the `js` directory we use `yarn` to add the Liquid Oxygen library.
 ```sh
 yarn init --yes
-yarn add @blueprintjs/core
+yarn add @emdgroup-liquid/liquid
 ```
 
-We will use a bundler to generate the `blueprint.js` script
+In order to use react components we need to find where package exports are defined first. We need to look for export keyword with names of components. In case of this package, exports can be found in `@emdgroup-liquid/liquid/dist/react`.
+
+We will use a bundler to generate the `liquid.js` script
 from the following `js/src/index.js` file:
 ```js
-const Blueprint = require('@blueprintjs/core');
+import Liquid from '@emdgroup-liquid/liquid/dist/react'
```

Comment:
I haven't noticed this earlier.... Later in the tutorial it's `const Liquid = require('@emdgroup-liquid/liquid/dist/react');`, we probably should use the same here
```suggestion
const Liquid = require('@emdgroup-liquid/liquid/dist/react');
```

Reply:
corrected